### ISSUE TITLE
Have a parameter for the year needed

### ISSUE THREAD

**Issue by CIOData** [pdil/usmap#110]
I was mapping county-level data today using usmap and realized that my values for Connecticut were being rejected. I am using data from 2016-2020, so it still has the older Connecticut counties in it, but apparently plot_usmap() only recognizes the new counties (which have different FIPS codes than the previous one). Would it be possible to allow a year argument so that the appropriate geometries get included for older data? I imagine this may come in useful in other circumstances as well (such as the ever evolving city/county issue in Virginia).

Thanks.

**Comment by pdil** [comment]
Yes this is a good idea, I will have to do an analysis of the file sizes involved since adding too many yearly map data sets may increase the package size beyond the limit that CRAN recommends (I think 5 MB for data). Currently the `usmapdata` files that `usmap` uses take up about 3 MB uncompressed (~1 MB compressed) so there is probably room to add a few more years but either way I will investigate further.

**Comment by CIOData** [comment]
Thanks. The Connecticut one is the most pressing. Changes in Virginia can be worked around in the data, but there is no direct translation of data between old and new Connecticut counties.

Sent via the Samsung Galaxy S22 Ultra 5G, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by pdil** [comment]
Support for multiple years of data is now available in `usmapdata`. I am now updating `usmap` to allow for use of these data sets.
